charts/stardog/files/utils.sh

```diff
@@ -32,7 +32,19 @@ function change_pw {
     PORT=${2}
     NEW_PW=$(cat /etc/stardog-password/adminpw)
     /opt/stardog/bin/stardog-admin --server http://${HOST}:${PORT} user passwd -N ${NEW_PW}
-    RC=$?
-    return ${RC}
+    if [[ $? -eq 0 ]];
+    then
+	    echo "Password successfully changed"
+	    return 0
+    else
+    	/opt/stardog/bin/stardog-admin --server http://${HOST}:${PORT} server status -u admin -p ${NEW_PW}
+    	if [[ $? -eq 0 ]];
```

Comment:
A non-zero error code doesn't mean auth failed, unfortunately. Let's use curl instead. That will fail with return code 22 if auth fails:
```
curl --fail -u admin:admin http://<host>:<port>/admin/status
curl: (22) The requested URL returned error: 401
```